pages/documentation/simulation/powerflow/index.md

```diff
@@ -119,34 +128,34 @@ The default value is `false`.
 The `writeSlackBus` is an optional property that says if the slack bus has to be written in the network using the [slack terminal extension](../../grid/model/extensions.md#slack-terminal) after a load flow computation.
 The default value is `false`.
 
-**noGeneratorReactiveLimits**  
-The `noGeneratorReactiveLimits` property is an optional property that defines whether the load flow is allowed to find a setpoint value outside the reactive limits of a generator or not.
-The default value is `false`.
+**useReactiveLimits**  
+The `useReactiveLimits` property is an optional property that defines whether the load flow should take into account equipment's reactive limits. Applies to generators, batteries, SVCs, dangling lines, and HVDC VSCs.
```

Comment:
Can you avoid using SVC? I think that using the complete static var compensators is better for documentation. But VSC converter station is okay.